src/backend/server/authentication/views.py

```diff
@@ -52,7 +52,7 @@ def post(self, request):
         # urrent_site = get_current_site(request).domain
         # relativeLink = reverse("email-verify")
         base_url = os.environ.get("SHORTLIST_API_URL")
-        absurl = base_url + "/auth/email-verify?token=" + str(token)
+        absurl = base_url + "auth/email-verify?token=" + str(token)
```

Comment:
Is this slash needed? I remember there was an issue once with a missing slash, not sure if this was resolved elsewhere.

Reply:
yeah, not needed. thats why undo it.